pkg/os/windows/powershell/powershell_windows.go

```diff
@@ -52,6 +52,7 @@ func IsAdmin() bool {
 
 func Execute(args ...string) (stdOut string, stdErr string, err error) {
 	args = append([]string{"-NoProfile", "-NonInteractive", "-ExecutionPolicy", "RemoteSigned", "-Command"}, args...)
+	/* #nosec */
```

Comment:
Did you audit all Execute callers for that G204 issue? Do you know if we can tell gosec to apply the G204 check to calls to Execute ?

Reply:
> Did you audit all Execute callers for that G204 issue?

Yes, same nothing comes from user side which we take and just execute.

```bash
$ git grep powershell.Execute
pkg/crc/preflight/preflight_checks_windows.go:  stdOut, _, _ := powershell.Execute(windowsReleaseId)
pkg/crc/preflight/preflight_checks_windows.go:  stdOut, _, _ := powershell.Execute(checkHypervisorPresent)
pkg/crc/preflight/preflight_checks_windows.go:  stdOut, _, _ = powershell.Execute(checkVmmsRunning)
pkg/crc/preflight/preflight_checks_windows.go:  _, _, err := powershell.ExecuteAsAdmin(enableHyperVCommand)
pkg/crc/preflight/preflight_checks_windows.go:  stdOut, _, _ := powershell.Execute(checkIfMemberOfHyperVAdmins)
pkg/crc/preflight/preflight_checks_windows.go:  outGroupName, _, err := powershell.Execute(`(New-Object System.Security.Principal.SecurityIdentifier("S-1-5-32-578")).Translate([System.Security.Principal.NTAccount]).Value`)
pkg/crc/preflight/preflight_checks_windows.go:  _, _, err = powershell.ExecuteAsAdmin(netCmdArgs)
pkg/crc/preflight/preflight_checks_windows.go:          _, stdErr, _ := powershell.Execute(checkIfDefaultSwitchExists)
pkg/crc/services/dns/dns_windows.go:    stdOut, _, _ := powershell.Execute(getDNSServerCommand)
pkg/crc/services/dns/dns_windows.go:    mainInterface, _, _ := powershell.Execute(getMainInterfaceCommand)
```

> Do you know if we can tell gosec to apply the G204 check to calls to Execute ?

No, I don't think it is  possible since the ruleset only check the syscalls https://github.com/securego/gosec/blob/master/rules/subproc.go#L57-L64